persistence/javadsl/src/main/java/com/lightbend/lagom/javadsl/persistence/PersistentEntityRef.java

```diff
@@ -74,6 +75,24 @@ public String entityId() {
         Patterns.ask(region, new CommandEnvelope(entityId, command), timeout);
 
     return future
+        .exceptionally(
+            cause -> {
+              if (cause instanceof AskTimeoutException) {
+                String msg =
+                    "Ask timed out on ["
+                        + this
+                        + "] after ["
+                        + timeout.toMillis()
+                        + " ms]. Message of type ["
```

Comment:
This is to really mimic the exception that Akka is creating. The timeout is always converted to `ms`.